src/pages/Home.js

```diff
@@ -20,6 +21,30 @@ const StyledSubline = Styled.p`
 `;
 
 
+const infoContent = [
+  {
+    title: 'How it works',
+    text: [
+      '• We find the 500 top posts from the past year for a subreddit.',
+      '• The data is visualized in a heatmap grouped by weekday and hour of the day.',
+      '• See immediately when to submit your reddit post.',
+    ],
+  },
+  {
+    title: 'About',
+    text: [
+      'This app was created during a course on & nbsp;'
```

Comment:
This doesn't seem to work (see the screenshot in the other comment). I would suggest passing a children prop instead of the text array. You could also use the `dangerouslySetInnerHTML` prop, but that's not very nice and should be restricted to limited use-cases.

Reply:
<Info title={infoContent[0].title}>
This app was created during a course on <StyledLink href = "https://www.ooloo.io" target = "_blank" rel ="noopener no-referrer">ooloo.io</StyledLink> with the goal to implement ...
</Info>